plugins/zoom-to-fit/src/index.ts

```diff
@@ -32,24 +32,24 @@ export class ZoomToFitControl implements Blockly.IPositionable {
   private top = 0;
   
   /**
-   * Width of the zoom-to-fit control.
+   * width of the zoom-to-fit control.
```

Comment:
Revert capitalization change, and in below docs for `height`

Reply:
Done!